queue_job/jobrunner/runner.py

```diff
@@ -219,16 +219,14 @@ def close(self):
 
     def _has_queue_job(self):
         with closing(self.conn.cursor()) as cr:
-            try:
-                cr.execute("SELECT 1 FROM ir_module_module "
-                           "WHERE name=%s AND state=%s",
-                           ('queue_job', 'installed'))
-            except psycopg2.ProgrammingError as err:
-                if unicode(err).startswith('relation "ir_module_module" '
-                                           'does not exist'):
-                    return False
-                else:
-                    raise
+            cr.execute("SELECT to_regclass('ir_module_module')")
+            [ir_module_module_exists] = cr.fetchone()
+            if not ir_module_module_exists:
+                return False
+            cr.execute(
+                "SELECT 1 FROM ir_module_module WHERE name=%s AND state=%s",
+                ('queue_job', 'installed')
+            )
```

Comment:
Same... I could find a comment speaking about 9.3 used with Odoo 10.0.

On Travis, the default version is 9.1 (https://github.com/OCA/maintainer-quality-tools/issues/432#issuecomment-281580935).

I think that Odoo 10.0 is still compatible with 9.0+ and we should aim for the same compatibility.